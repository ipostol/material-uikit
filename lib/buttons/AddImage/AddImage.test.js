'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _enzyme = require('enzyme');

var _AddImage = require('./AddImage');

var _AddImage2 = _interopRequireDefault(_AddImage);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

describe('AddImage component', function () {

  it('should render correctly', function () {

    var wrapper = (0, _enzyme.shallow)(_react2.default.createElement(
      _AddImage2.default,
      null,
      'AddImage'
    ));

    expect(wrapper).toMatchSnapshot();
  });
});