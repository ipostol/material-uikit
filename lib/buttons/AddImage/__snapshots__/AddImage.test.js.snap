// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddImage component should render correctly 1`] = `
<Button
  className="addImageButton l"
  element="button"
  icon={<UploadSmall />}
  mod="link"
  onClick={[Function]}
  round="small"
>
  AddImage
</Button>
`;
